### PR TITLE
added shadow dom documentation to finders in all translations #1070

### PR DESCRIPTION
**Pull request by shbenzer**
### **User description**
Fixed Bug #1070 

### Description
added shadow dom documentation to finders in all translations, thereby fixing Bug #1070 

### Motivation and Context
make documentation more comprehensive

### Types of changes

- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

___

### **PR Type**
Documentation

___

### **Description**
- Added comprehensive documentation for evaluating the Shadow DOM across multiple languages (English, Japanese, Portuguese, and Chinese).
- Included code examples for Java, Python, CSharp, and Ruby to demonstrate accessing the Shadow DOM.
- Highlighted the requirement for Selenium 4.0 or greater to use the shadow root methods.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>finders.en.md</strong><dd><code>Add Shadow DOM documentation and examples in English</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/elements/finders.en.md

<li>Added documentation for evaluating the Shadow DOM.<br> <li> Included code examples for Java, Python, CSharp, and Ruby.<br> <li> Highlighted the requirement for Selenium 4.0 or greater.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1887/files#diff-e21fb1eabd6a7eaf605f8f733ad4e5406d5ad60908a37a6da9b2c363c021ceae">+32/-0</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>finders.ja.md</strong><dd><code>Add Shadow DOM documentation and examples in Japanese</code>&nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/elements/finders.ja.md

<li>Added Shadow DOM documentation in Japanese.<br> <li> Provided code examples for multiple programming languages.<br> <li> Mentioned Selenium version requirement.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1887/files#diff-c047d3fd7654069cb4ca13dcf34df864532041dd9a03abced9f295402313e42c">+32/-0</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>finders.pt-br.md</strong><dd><code>Add Shadow DOM documentation and examples in Portuguese</code>&nbsp; &nbsp; </dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/elements/finders.pt-br.md

<li>Introduced Shadow DOM documentation in Portuguese.<br> <li> Included examples for Java, Python, CSharp, and Ruby.<br> <li> Noted the need for Selenium 4.0 or greater.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1887/files#diff-eaadee9340583f10ee129591af6c74011a61c7535fb37bbd08c77bd6afaae437">+32/-0</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>finders.zh-cn.md</strong><dd><code>Add Shadow DOM documentation and examples in Chinese</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/elements/finders.zh-cn.md

<li>Added Shadow DOM documentation in Chinese.<br> <li> Provided code examples for several languages.<br> <li> Highlighted Selenium version requirement.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1887/files#diff-eab325a79563cf7c8c85aecf2de106c1cf35ff7b191245b1cf7eac5cfed1d6de">+32/-0</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

**Comment by shbenzer** [comment]
thank you @titusfortner for the great blog reference!

File: website_and_docs/content/documentation/webdriver/elements/finders.en.md [hunk 1]
```diff
@@ -108,6 +108,38 @@ driver implementation supports a given feature. These interfaces are clearly def
 to adhere to having only a single role of responsibility.
 {{% /pageinfo %}}
 
+### Evaluating the Shadow DOM
+
+The Shadow DOM is an encapsulated DOM tree hidden inside an element. With the release of v96 in Chromium Browsers, Selenium can now allow you to access this tree with easy-to-use shadow root methods. NOTE: These methods require Selenium 4.0 or greater.
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/elements/finders.en.md]: HI @shbenzer ,

Thank you for the PR.

Could you please limit each line to 100 characters or fewer?
see __Line Length__ in here https://www.selenium.dev/documentation/about/style/

**Review by harsha509 — CHANGES_REQUESTED** [review]

File: website_and_docs/content/documentation/webdriver/elements/finders.en.md (hunk 1 above)
Comment (shbenzer) [reply website_and_docs/content/documentation/webdriver/elements/finders.en.md]: My bad, I fixed it.

**Review by harsha509 — APPROVED** [review]
Thank you @shbenzer !

**Pull request merged**